packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -138,15 +137,11 @@ export function BaseIModelPage(props: BaseIModelProps) {
     }));
   };
 
-  const isPropertyInvalid = (value: string) => {
-    return value.length > MAX_LENGTH;
-  };
-
   const isDataValid = () => {
     return (
       !!imodel.name.length &&
-      !isPropertyInvalid(imodel.name) &&
-      !isPropertyInvalid(imodel.description) &&
+      !isPropertyInvalid(imodel.name, MAX_LENGTH) &&
```

Comment:
you still ahve multiple max length defnitions....... use one please and better name with capital letters and underscore so its clear its constant

Reply:
fixed